test/GLMTest.m

```diff
@@ -22,6 +22,178 @@ function tear_down(testCase)
+            
+            % Generate a new file name in the same folder
+            dHbFile = bst_process('GetNewFilename', bst_fileparts(sStudy.FileName), 'data_dHb');
+            sDataOut.FileName = file_short(dHbFile);
+            bst_save(dHbFile, sDataOut, 'v7');
+            db_add_data(iStudy, dHbFile, sDataOut);
+
+            sHb = bst_process('GetInputStruct', dHbFile);
+            
+            
+            sHb_filtered = bst_process('CallProcess', 'process_bandpass', sHb, [], ...
+                'sensortypes', '', ...
+                'highpass',    0.01, ...
+                'lowpass',     0.15, ...
+                'tranband',    0, ...
+                'attenuation', 'strict', ...  % 60dB
+                'ver',         '2019', ...  % 2019
+                'mirror',      0, ...
+                'overwrite',   0);
+            
+            sGlmResults = bst_process('CallProcess','process_nst_glm_fit', sHb_filtered, [], ...
+                                                    'hpf_low_cutoff',         0.01, ...
+                                                    'trim_start',             0, ...
+                                                    'fitting',                1, ...  % OLS
+                                                    'statistical_processing', method_test, ...
+                                                    'noise_model',            1, ...  % AR(1)
+                                                    'stim_events',            strjoin(names, ','), ...
+                                                    'hrf_model',              1, ...  % CANONICAL
+                                                    'trend',                  1, ...
+                                                    'save_betas',             0, ...
+                                                    'save_residuals',         0, ...
+                                                    'save_fit',               0);
+                
+        
+             % Check activation detection (p-val thresholding)
+            con_sResults_stim1 = bst_process('CallProcess', 'process_nst_glm_contrast', sGlmResults, [], ...
+                                                  'Contrast', '[1 -1]');
+            stat_sResults_stim1 = bst_process('CallProcess', 'process_nst_glm_contrast_ttest', ...
+                                                  con_sResults_stim1, [], ...
+                                                  'tail', 'two');
+            glm_results_stat = in_bst_matrix(stat_sResults_stim1.FileName);
+            thresh_options.pThreshold = 5e-2;
+            thresh_options.Correction = 'bonferroni';
+            thresh_options.Control = 1;
+            
+            [glm_results, corr_p] = bst_stat_thresh(glm_results_stat.pmap, thresh_options);
+            true_map=zeros(1,n_chan);
+            true_map(active_channels)=1;
+            
+            plot_performance(glm_results_stat,true_map);
```

Comment:
It's better not to plot anything by default in unit tests.
You can add a "if 0" that you change to "if 1" when you need the plots while debugging